airbyte-commons/src/main/java/io/airbyte/commons/lang/Exceptions.java

```diff
@@ -39,6 +39,16 @@ public static void toRuntime(final Procedure voidCallable) {
     castCheckedToRuntime(voidCallable, RuntimeException::new);
   }
 
+  public static Runnable toSwallowExceptionRunnable(final Procedure voidCallable) {
+    return () -> {
+      try {
+        voidCallable.call();
+      } catch (Exception e) {
+        e.printStackTrace();
```

Comment:
This should probably use a logger instead of just printing to standard out.  I believe that this will get flagged by the static analysis tools that I am adding.  However, if there is a good reason to do this, we can always add an override to ignore this when/if we get to improved static analysis linting.

Reply:
Good call!